contributing_guide.md

```diff
@@ -0,0 +1,37 @@
+## Contributing code commits
```

Comment:
The file is usually named CONTRIBUTING.md. eg: https://github.com/rails/rails/blob/master/CONTRIBUTING.md

We should place it in the .github directory